tutorials/nlp/Relation_Extraction-BioMegatron.ipynb

```diff
@@ -0,0 +1,601 @@
+    "import wget \n",
+    "import torch\n",
+    "import pytorch_lightning as pl\n",
+    "from omegaconf import OmegaConf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "daYw_Xll2ZR9"
+   },
+   "source": [
+    "In this tutorial, we are going to describe how to finetune a BERT-like model based on [BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding](https://arxiv.org/abs/1810.04805) on [RE: Text mining chemical-protein interactions (CHEMPROT)](https://biocreative.bioinformatics.udel.edu/tasks/biocreative-vi/track-5/). "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "ZnuziSwJ1yEB"
+   },
+   "source": [
+    "# Datasets\n",
+    "\n",
+    "Details of ChemProt Relation Extraction task and the original data can be found on the [BioCreative VI website](https://biocreative.bioinformatics.udel.edu/tasks/biocreative-vi/track-5/)\n",
+    "\n",
+    "ChemProt dataset pre-processed for easier consumption can be downloaded from [here](https://github.com/arwhirang/recursive_chemprot/blob/master/Demo/tree_LSTM/data/chemprot-data_treeLSTM.zip) or [here](https://github.com/ncbi-nlp/BLUE_Benchmark/releases/download/0.1/bert_data.zip)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "--wJ2891aIIE"
+   },
+   "outputs": [],
+   "source": [
+    "# supported task names: [\"cola\", \"sst-2\", \"mrpc\", \"sts-b\", \"qqp\", \"mnli\", \"qnli\", \"rte\", \"wnli\"]\n",
+    "TASK = 'ChemProt'\n",
+    "DATA_DIR = 'DATA_DIR'\n",
+    "WORK_DIR = 'WORK_DIR'\n",
+    "MODEL_CONFIG = 'text_classification_config.yaml'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "os.makedirs(DATA_DIR, exist_ok=True)\n",
+    "os.makedirs(os.path.join(DATA_DIR, 'RE'), exist_ok=True)\n",
+    "os.makedirs(WORK_DIR, exist_ok=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# download the dataset\n",
+    "wget.download('https://github.com/arwhirang/recursive_chemprot/blob/master/Demo/tree_LSTM/data/chemprot-data_treeLSTM.zip?raw=true',\n",
+    "              os.path.join(DATA_DIR, 'data_re.zip'))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!unzip DATA_DIR/data_re.zip -d DATA_DIR/RE"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "qB0oLE4R9EhJ"
+   },
+   "outputs": [],
+   "source": [
+    "! ls -l $DATA_DIR/RE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Pre-process dataset\n",
+    "Let's convert the dataset into NeMo compatible format"
```

Comment:
What is this format? Please describe or add a link to the description. You can refer to the text_classification notebook - although no link is available but it's WIP, so maybe just mention Text Classification notebook and add a link to tutorials/nlp - ?